server/views/case-summary-record-order.njk

```diff
@@ -134,9 +135,9 @@
 
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-one-third">
-                    <p class="govuk-hint govuk-!-margin-bottom-0 qa-start-title">{{ "Post-release status" if custodialOrder else "Order started" }}</p>
+                    <p class="govuk-hint govuk-!-margin-bottom-0 qa-start-title">{{ "Post-release status" if custodialOrder and not inCustody else "Order started" }}</p>
                     <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-0 qa-start-date">
-                        {%- if custodialOrder -%}
+                        {%- if custodialOrder and not inCustody -%}
```

Comment:
`inCustody` is only ever used in this combination  
```if custodialOrder and not inCustody```

`custodialOrder and not inCustody` seems like a good candidate for  variable  itself, something like `hasLeftCustody`

Reply:
I was going to use the `inCustody` variable further down the page where we use the expression `if custodialType === "In Custody"`,  but forgot to update that but good point.